cfgov/v1/atomic_elements/organisms.py

```diff
@@ -6,6 +6,13 @@
 from ..util import ref
 from ..models.snippets import Contact as ContactSnippetClass
 
+JS_ORGANISMS = [
```

Comment:
Why not make this a list of types instead of strings?

below you define page handlers:

`PAGE_HANDLERS = [JSHandler]`